### DISCUSSION TITLE
exportExcel or exportPPT not working for me on Windows

### DISCUSSION THREAD

**Discussion by ayanb2312** [docToolchain/docToolchain#1110] [Q&A]
I am new to DocToolchain and was trying to play with its tasks like exportExcel or exportPPT on my windows machine. Based on the resources available in internet, I could setup DTCW and generate the arc42 template based documentation in HTML/PDF. But when I am attempting exportExcel or exportPPT tasks, it is giving me below error:

* What went wrong:
Task './src/docs/excels/ExcelTest.xlsx' not found in root project 'docToolchain'.

Can anyone point me if I need to do some additional configuration in docToolchainConfig.groovy file or what I am missing here?

**Comment by rdmueller** [comment]
hm. that is interesting. ExportExcel is a task with which we have nearly no problems.

if you have a basic doctoolchain setup (see https://doctoolchain.org/docToolchain/v2.0.x/020_tutorial/010_Install.html), you should be able to place an excel file in `src/docs/.` and execute `./dtcw exportExcel` as a result, the exported excel content should show up in `src/excel/[filename]/[worksheet].(adoc|cvs)`.

your error message look like you tried to execute the given excel file as a task. Did you maybe execute `./dtcw exportExcel ./src/docs/excels/ExcelTest.xlsx`? This is not necessary. docToolchain will search for all existing Excel files in `./src/docs`. On the command line, you only specify only the tasks to be executed. Details for other tasks are specified in the `docToolchainConfig.groovy` file.

Does this help?
